gammapy/image/models/core.py

```diff
@@ -105,9 +122,10 @@ def __init__(self, lon_0, lat_0, sigma):
     def evaluate(lon, lat, lon_0, lat_0, sigma):
         """Evaluate the model (static function)."""
         sep = angular_separation(lon, lat, lon_0, lat_0)
-        norm = 1 / (2 * np.pi * sigma ** 2)
-        exponent = -0.5 * (sep / sigma) ** 2
-        return norm * np.exp(exponent)
+        a = 1.0 - np.cos(sigma)
+        norm = (1 / (4 * np.pi * a * (1.0 - np.exp(-1.0 / a)))) * u.sr ** -1
```

Comment:
The term `(...) * u,sr ** -1` creates a new quantity object every time the method is evaluated. By default all the data will be copied, which slows down the computation a lot. Please remove it here. Here is some info about this http://docs.astropy.org/en/stable/units/index.html#astropy-units-performance